### PR TITLE
Add bulk typo fixes to the hp-edit.obo file specifically

### PR DESCRIPTION
**Pull request by cmdcolin**
follow up to comment in https://github.com/monarch-initiative/mondo/pull/7685#issuecomment-2096474860

my workflow to make these fixes is to run typos-cli and then run ruplacer on each suggestion manually (which is a little laborious and is a little error prone but works ok :))

let me know if you have any suggestions. the _typos.toml, which ignores some known false positives, is included in the pr, and you can even consider running the typos tool in continuous integration/github action (i do this on source code repos)

### CODE REVIEW

**Comment by matentzn** [comment]
@cmdcolin This is amazing, thank you so much.

@LCCarmody can you review this PR? When you are done, tell me and I will merge (I am not sure yet if I want to include the toml file).

File: src/ontology/hp-edit.owl [hunk 1]
```diff
@@ -154860,7 +154860,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0030778> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_0030779> (Ethmocephaly)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0002-0003-6754>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:23248551") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0030779> "Ethmocephaly is the rarest form of holoprosencephaly, which occurs due to an incomplete cleavage of the forebrain. Clinically, the disease presents with a proboscis, hypotelorism, microphthalmos and malformed ears.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0002-0003-6754>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:23248551") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0030779> "Ethmocephaly is the rarrest form of holoprosencephaly, which occurs due to an incomplete cleavage of the forebrain. Clinically, the disease presents with a proboscis, hypotelorism, microphthalmos and malformed ears.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:154863]: This correction is incorrect

File: src/ontology/hp-edit.owl [hunk 2]
```diff
@@ -155867,28 +155867,28 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0030889> <http://purl.obolibrary.o
...
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0030891> "Periventricular white matter hyperintensities")
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0030891> <http://purl.obolibrary.org/obo/HP_0002518>)
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0030891> <http://purl.obolibrary.org/obo/HP_0030890>)
 
 # Class: <http://purl.obolibrary.org/obo/HP_0030892> (Deep cerebral white matter hyperintensities)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:15576652") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0030892> "Areas of brighter than expected signal on magnetic resonance imaging emanating from locations distant from the ventricular system.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:15576652") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0030892> "Areas of brigther than expected signal on magnetic resonance imaging emanating from locations distant from the ventricular system.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:155888]: This is incorrect. It should be original

File: src/ontology/hp-edit.owl [hunk 3]
```diff
@@ -159254,7 +159254,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0031296> <http://purl.obolibrary.o
...
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:19398672") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0031297> "Unroofed coronary sinus (CS) is a rare congenital cardiac anomaly in which there is partial (either focal or fenestrated) or complete absence of the roof of the CS, which results in a communication between the CS and the LA. Unroofed CS is the rarrest type of atrial septal defect. It is often associated with persistent left superior vena cava (LSVC) and other forms of complex congenital heart disease, usually heterotaxia syndromes. The morphological types have been classified into 4 groups: Type I, completely unroofed with persistent LSVC; type II, completely unroofed without persistent LSVC; type III, partially unroofed mid portion; and type IV, partially unroofed terminal portion.")
 AnnotationAssertion(dc:date <http://purl.obolibrary.org/obo/HP_0031297> "2017-08-13T21:44:35Z"^^xsd:dateTime)
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:159258]: Original is correct.

File: src/ontology/hp-edit.owl [hunk 4]
```diff
@@ -161749,7 +161749,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0031574> <http://purl.obolibrary.o
...
...en the alar base and the cleft of the medial aspect of the lower eyelid. The inferior lacrimal punctum is evident at the lateral margin of the lower eyelid cleft. The lacrimal drainage system ends as an opening directly onto the cheek without communication into the nasal cavity. The globe is normal in size, but it is displaced inferiorly and laterally. The nasal septum shows the characteristic distortion seen in typical cleft lip and palate. There is absence of septation between the nasal cavity on the cleft side and the maxilla. The maxilla is hypoplastic in three dimensions, with a marked reduction in pneumatization. Superior extension of the skeletal clefting into the medial portion of the orbital floor and into the inferior orbital rim in the region of the frontal process of the maxilla allows direct communication between the orbit above and the nasomaxillary region below. There is mild narrowing of the ethmoid sinus and of the body of the sphenoid on the cleft side. The pterygoid process appears anatomically normal, but less displaced from the midline compared with that of the noncleft side. Both the orbit and the floor of the anterior cranial fossa are inferiorly displaced.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:161752]: It was hard to find the differences in this, but the one correction I found is incorrect. It should be the original 'disrupting

File: src/ontology/hp-edit.owl [hunk 5]
```diff
@@ -173752,7 +173752,7 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(dcterms:creator <http://purl.obolibrary.org/obo/HP_0032988> <https://orcid.org/0000-0002-0736-9199>)
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/HP_0032988> "Head lag")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/HP_0032988> "Headlag")
-AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0032988> "During pull-to-sit, the normal response is for an infant to right the head and maintain it in line with the shoulders. Head lag is demonstrated when the head is not righted but lags posteriorly behind the trunk as a result of poor head and neck control.")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0032988> "During pull-to-sit, the normal response is for an infant to right the head and maintain it in line with the shoulders. Head lag is demonstrated when the head is not rigthed but lags posteriorly behind the trunk as a result of poor head and neck control.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:173755]: Original was correct

File: src/ontology/hp-edit.owl [hunk 6]
```diff
@@ -174193,7 +174193,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0033040> <http://purl.obolibrary.o
...
...racrine, and/or endocrine activity and, through receptor binding, can elicit a variety of responses, depending upon the cytokine and the target cell. Among the many functions of cytokines are the control of cell proliferation and differentiation and the regulation of angiogenesis and immune and inflammatory responses. Inflammation associated with a cytokine storm begins at a local site and spreads throughout the body via the systemic circulation. Rubor (redness), tumor (swelling or edema), calor (heat), dolor (pain), and function laesa (loss of function) are the hallmarks of acute inflammation. While the general concept of an excessive or uncontrolled release of proinflammatory cytokines is well known, a precise definition of what constitutes a cytokine storm is lacking. The first use of cytokine storm appears to be in an article published in 1993 on graft-versus-host disease. Cytokine storm has subsequently been described in patients infected by cytomegalovirus, Epstein-Barr virus-associated hemophagocytic lymphohistiocytosis, group A streptococcus, influenza virus, variola virus, severe acute respiratory syndrome coronavirus (SARS-CoV),avian H5N1 influenza virus, and SARS-CoV-2")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:174196]: Not sure what changed between these 2, but a space was missed before avian in last line.

File: src/ontology/hp-edit.owl [hunk 7]
```diff
@@ -174728,7 +174728,7 @@ AnnotationAssertion(dc:date <http://purl.obolibrary.org/obo/HP_0033106> "2020-08
...
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0033106> "The D-dimer antigen is a unique marker of fibrin degradation that is formed by the sequential action of 3 enzymes: thrombin, factor XIIIa, and plasmin. First, thrombin cleaves fibrinogen producing fibrin monomers, which polymerize and serve as a template for factor XIIIa and plasmin formation. Second, thrombin activates plasma factor XIII bound to fibrin polymers to produce the active transglutaminase, factor XIIIa. Factor XIIIa catalyzes the formation of covalent bonds between D-domains in the polymerized fibrin. Finally, plasmin degrades the crosslinked fibrin to release fibrin degradation products and expose the D-dimer antigen. D-dimer measurement has been most comprehensively validated in (1) the exclusion of venous thromboembolism (VTE) in certain patient populations and (2) the diagnosis and monitoring of coagulation activation in disseminated intravascular coagulation. Abnormal D-dimer in the appropriate clinical context denotes that there is no major ongoing activation of intravascular coagulation, and serves as a reliable tool for the exclusion of VTE.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:174731]: Not sure what was changed

File: src/ontology/hp-edit.owl [hunk 8]
```diff
@@ -174872,7 +174872,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0033121> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_0033122> (Absent P wave)
 
-AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0033122> "The P wave that normally precedes each QRS complex by a fixed PR interval of 120 to 200 milliseconds is not present.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0033122> "The P wave that normally precedees each QRS complex by a fixed PR interval of 120 to 200 milliseconds is not present.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl]: original was correct

File: src/ontology/hp-edit.owl [hunk 9]
```diff
@@ -174944,7 +174944,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0033129> <http://purl.obolibrary.o
...
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0033130> "For diagnostic ultrasonography, high-frequency sound waves are generated and received by the ultrasonography transducer, which is placed on the skin. Returning sound waves (echoes) are processed by a computer and displayed on a computer screen. A gray-scale image is produced when the ultrasonography machine operates in B-mode, or brightness mode, in which returning echoes are represented as bright dots; the brightness of the dots represents the strength of the reflected echoes. Echogenicity, therefore, refers to how bright or dark something appears in the gray-scale image; the brigther something appears, the more echogenic it is. With regard to the kidney, echogenicity generally refers to how bright or dark the kidney parenchyma appears in comparison to the liver.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl]: original was correct.

File: src/ontology/hp-edit.owl [hunk 10]
```diff
@@ -177109,7 +177109,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0033376> <http://purl.obolibrary.o
...
...org/0000-0003-0113-912X>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:20884725") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:26474448") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:30521210") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:32558323") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:33306420") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0033377> "Presence of increased numbers of bombesin-immuno-positive neuroendocrine cells (NECs) within distal airways. There are no formal criteria for an increase (which is also dependent on the sensitivity of the staining method), findings of neuroendocrine cells in at least 70% of bronchioles by lung biopsy and at least 10% NECs in an individual airway are consistent with the diagnosis of Neuroendocrine cell hyperplasia of infancy in the appropriate clinical setting. Increases are also seen in other clicnial settings. Neuroendocrine neoplasms of the lung encompass NE tumors (NETs), which split into typical and atypical carcinoids, and NE carcinomas (NECs).")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:177112]: one change was correct, but missed 'clicnial'

File: src/ontology/hp-edit.owl (hunk 7 above)
Comment (cmdcolin) [reply src/ontology/hp-edit.owl:174731]: Anormal->Abnormal

File: src/ontology/hp-edit.owl [hunk 11]
```diff
@@ -178168,7 +178168,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0033497> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_0033498> (Segmental glomerulosclerosis away from the vascular and tubular poles)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0003-2356-950X>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0033498> "Segmental solidification of the glomerular tuft characterized by increased extracellular matrix, causing glomerular capillary obliteration at neirhter the tubular or vascular poles. Features including hyalinosis, foam cells, hypertrophy of overlying glomerular epithelial cells can also be observed.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0003-2356-950X>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0033498> "Segmental solidification of the glomerular tuft characterized by increased extracellular matrix, causing glomerular capillary obliteration at neirther the tubular or vascular poles. Features including hyalinosis, foam cells, hypertrophy of overlying glomerular epithelial cells can also be observed.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:178171]: neither is correct...which is a little ironic here.

File: src/ontology/hp-edit.owl (hunk 1 above)
Comment (cmdcolin) [reply src/ontology/hp-edit.owl]: thanks for catching, there were some manual steps that caused some errors like this. should be fixed now :+1:

File: src/ontology/hp-edit.owl [hunk 12]
```diff
@@ -183405,16 +183405,16 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0034071> <http://purl.obolibrary.o
...
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:21472065") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0034072> "Any deviation from the normal appearing echogenicity of the fetal bone. Echogenicity refers to the ability of tissue to reflect ultrasound waves back toward the transducer and produce an echo. Whenever there is an interface of structures with different echogenicities, a visible difference in contrast will be apparent on the screen. The higher the echogenicity of tissues, the brigther they appear on ultrasound imaging.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl]: original was correct

File: src/ontology/hp-edit.owl (hunk 3 above)
Comment (cmdcolin) [reply src/ontology/hp-edit.owl]: should be fixed :+1:

File: src/ontology/hp-edit.owl [hunk 13]
```diff
@@ -183405,16 +183405,16 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0034071> <http://purl.obolibrary.o
...
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0034072> "Hyperechoic structures are seen as brigther on the US image relative to surrounding structures due to higher reflectivity of the US beam. Isoechoic structures are seen as bright as surrounding structures on conventional US imaging due to similar reflectivity to the US beam. Hypoechoic structures are seen as darker relative to the surrounding structures on the US image due to the US beam being reflected to a lesser extent. Anechoic structures that lack internal reflectors fail to reflect the US beam to the transducer and are seen as homogeneously black on imaging.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl]: original was correct

File: src/ontology/hp-edit.owl (hunk 6 above)
Comment (cmdcolin) [reply src/ontology/hp-edit.owl:174196]: functio->function

added space too :+1:

File: src/ontology/hp-edit.owl (hunk 8 above)
Comment (cmdcolin) [reply src/ontology/hp-edit.owl]: should be fixed now :) thanks for catching

File: src/ontology/hp-edit.owl [hunk 14]
```diff
@@ -187959,15 +187959,15 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0034586> <http://purl.obolibrary.o
...
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:32034785") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0034587> "Nevus psiloliparus is a rare fatty tissue nevus that is a marker for encephalocraniocutaneous lipomatosis, a neurocutaneous syndrome with ocular and central nervous system anomalies. Clinically, nevus psiloliparus is often described as a congenital alopecia and appears as an irregularly shaped, circumscribed area of alopecia on the scalp. Histopathology demonstrates a near-complete absence of mature hair follicles with preservation of arector pili muscles and mature adipocytes within the dermis.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:187962]: original was correct

File: src/ontology/hp-edit.owl (hunk 10 above)
Comment (cmdcolin) [reply src/ontology/hp-edit.owl:177112]: good catch, I can propose this fix upstream to typos-cli :+1:. fixed in this pr too now

File: src/ontology/hp-edit.owl [hunk 15]
```diff
@@ -187959,15 +187959,15 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0034586> <http://purl.obolibrary.o
...
 SubClassOf(<http://purl.obolibrary.org/obo/HP_0034587> <http://purl.obolibrary.org/obo/HP_0003764>)
 
 # Class: <http://purl.obolibrary.org/obo/HP_0034588> (Crypt hyperplasia)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:17021129") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0034588> "Crypt hyperplasia denotes elongation of the length of the crypts of Lieberkuhn, a process that initially precedes villous atrophy. Elongation may be caused by expansion of the lamina propria as a result of the proliferation of stromal cells, an influx of inflammatory cells and tissue remodeling.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:17021129") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0034588> "Crypt hyperplasia denotes elongation of the length of the crypts of Lieberkuhn, a process that initially precedees villous atrophy. Elongation may be caused by expansion of the lamina propria as a result of the proliferation of stromal cells, an influx of inflammatory cells and tissue remodeling.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl]: original was correct

File: src/ontology/hp-edit.owl [hunk 16]
```diff
@@ -194643,7 +194643,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0040329> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_0040330> (Confluent hyperintensity of cerebral white matter on MRI)
 
-AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:25636970") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0040330> "Areas of brighter than expected MRI signal in the white matter of the brain whereby individual patches run together.")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "PMID:25636970") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0040330> "Areas of brigther than expected MRI signal in the white matter of the brain whereby individual patches run together.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl]: original was correct

File: src/ontology/hp-edit.owl [hunk 17]
```diff
@@ -196955,7 +196955,7 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date>
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/HP_0100005> "NCIT:C3234")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.obolibrary.org/obo/HP_0100005> "UMLS:C4022391")
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/HP_0100005> "HP:0100005")
-AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0100005> "Testicular mesothelioma originates from cells of the tunica vaginalis (the thin layer of mesothelium surrounding the testicles). Testicular mesothelioma is the rarest form of mesothelioma.")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0100005> "Testicular mesothelioma originates from cells of the tunica vaginalis (the thin layer of mesothelium surrounding the testicles). Testicular mesothelioma is the rarrest form of mesothelioma.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl]: original was correct

File: src/ontology/hp-edit.owl [hunk 18]
```diff
@@ -218989,7 +218989,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_4000143> <http://purl.obolibrary.o
 
 # Class: <http://purl.obolibrary.org/obo/HP_4000144> (Recent past medical history)
 
-AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_4000144> "History of events that precede the current presenting condition and are thought to be of potential relevance, having occurred in the hours, days, or weeks before the current condition.")
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_4000144> "History of events that precedee the current presenting condition and are thought to be of potential relevance, having occurred in the hours, days, or weeks before the current condition.")
```
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:218992]: original was correct

File: src/ontology/hp-edit.owl
Comment (LCCarmody) [inline src/ontology/hp-edit.owl:1]: I see that you got my comments. AFAIK I caught them all, but careful to make sure that what you are changing to is correct.

File: src/ontology/hp-edit.owl (hunk 14 above)
Comment (cmdcolin) [reply src/ontology/hp-edit.owl]: thanks, back to arrector

File: src/ontology/hp-edit.owl
Comment (cmdcolin) [reply src/ontology/hp-edit.owl:1]: thank you, i should have manually checked them myself! I think it should be OK now

Comment (LCCarmody) [reply src/ontology/hp-edit.owl:1]: Okay. Thanks for correcting.

**Review by LCCarmody — APPROVED** [review]

File: src/ontology/hp-edit.owl [hunk 19]
```diff
@@ -88606,7 +88606,7 @@ SubClassOf(<http://purl.obolibrary.org/obo/HP_0008678> <http://purl.obolibrary.o
...
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0002-0736-9199>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0008682> "Coagulative necrosis of tubular epithelial cells, defined as cells with increased cytoplasmic eosinophilia and nucleus that has a condensed chromatin pattern with fuzzy nuclear contour or has barrely visible nuclear basophilic staining. The extent of cortical tubular necrosis is scoredsemiquantitatively as none, mild (less than 25% tubules with necrosis), moderate (25-50 percent), and severe (over 50%).")
```
Comment (matentzn) [inline src/ontology/hp-edit.owl:88609]: @LCCarmody can you check this one again?

**Review by matentzn — APPROVED** [review]

File: src/ontology/hp-edit.owl (hunk 19 above)
Comment (LCCarmody) [reply src/ontology/hp-edit.owl:88609]: original is correct

Comment (matentzn) [reply src/ontology/hp-edit.owl:88609]:
```suggestion
AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://orcid.org/0000-0002-0736-9199>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/HP_0008682> "Coagulative necrosis of tubular epithelial cells, defined as cells with increased cytoplasmic eosinophilia and nucleus that has a condensed chromatin pattern with fuzzy nuclear contour or has barely visible nuclear basophilic staining. The extent of cortical tubular necrosis is scoredsemiquantitatively as none, mild (less than 25% tubules with necrosis), moderate (25-50 percent), and severe (over 50%).")
```

**Comment by matentzn** [comment]
@cmdcolin this is amazing, very thankful!

@LCCarmody thanks a ton for reviewing this carefully! 250 typos fixed in HPO! Much appreciated, also on behalf of the "Search" and "NLP" communities!

If you want to fix other resources reach out to me (linkedin or whatever you prefer).

**Pull request merged**